src/css/landing-page.css

```diff
@@ -268,36 +268,6 @@
   }
 }
 
-/* CSS for top section: H1, H2, and Capella Stars image */
-.doc.landing-page-capella .top-row {
-  display: flex;
-  flex-direction: row;
-  padding: 1rem;
-  margin-bottom: 2rem;
-  margin-top: -1rem;
-}
-
-.doc.landing-page-capella .top-row > .positioned-image {
-  flex-direction: row;
-  width: 35%;
-  min-width: 300px;
-  height: auto;
-  position: relative;
-  margin-top: -2.5rem; /* Adjust to position image right below the div */
-  margin-left: 1rem; /* Adjust to position image to the right of the div */
-}
-
-.doc.landing-page-capella .top-row > .top-row-text-container {
-  flex-direction: column;
-  padding-right: 1rem;
-  position: relative;
-}
-
-.doc.landing-page-capella .top-row > .top-row-text-container > h1 {
-  font-weight: 600;
-  margin-top: 1rem;
-}
```

Comment:
I had all of this in here to control the positioning of the stars and the text in relation. How are we going to display that image with none of these controls?